doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -294,13 +294,16 @@ detail`` returns a message similar to the following::
   bug you raise. See `Preparing your logs`_ for information about the proper
   preparation of logs.
 
-
-**What if state is ``leader`` or ``peon``?**
-
-  This should not happen:  famous last words.  If it does, however, it likely
-  has a lot to do with clock skew -- see `Clock Skews`_. If you are not
-  suffering from clock skew, then please prepare your logs (see
-  `Preparing your logs`_) and reach out to the community.
+**What does it mean when the Monitor state is ``leader`` or ``peon``?**
+
+  If ``ceph health detail`` shows that the Monitor is in the ``leader`` state
+  or in the ``peon`` state, it is likely that clock skew is present. Follow the
+  instructions in `Clock Skews`_. If you have followed the instructions in the
+  `Clock Skews`_ section, and ``ceph health detail`` still shows that the
```

Comment:
suggest "If you have followed those instructions, and"

Reply:
Accepted.